client/src/GlobalStyles.js

```diff
@@ -43,8 +43,8 @@ h2 {
   font-size: 1.5rem;
   color: var(--primary-color);
 }
-li {
-  display:flex;
+li { 
+  list-style-position:inside;
```

Comment:
Nice, didn't know this attribute 👍

Reply:
> Lgtm, nice work 🙂👍 the deployment link didn't work for me

Deployment is working now. forgot to add environments to heroku 🤦‍♀️